server/src/main/java/org/opensearch/threadpool/ThreadPool.java

```diff
@@ -261,6 +263,10 @@ public ThreadPool(
         );
         builders.put(Names.TRANSLOG_SYNC, new FixedExecutorBuilder(settings, Names.TRANSLOG_SYNC, allocatedProcessors * 4, 10000));
         builders.put(Names.REMOTE_PURGE, new ScalingExecutorBuilder(Names.REMOTE_PURGE, 1, halfProcMaxAt5, TimeValue.timeValueMinutes(5)));
+        builders.put(
+            Names.REMOTE_REFRESH,
+            new ScalingExecutorBuilder(Names.REMOTE_REFRESH, 1, allocatedProcessors * 4, TimeValue.timeValueMinutes(5))
```

Comment:
Basis follow up performance test, this will be tuned if required.

Reply:
Unless we have strong reasons, let's start with smaller threadpool and increase the size based on threadpool queue observed during perf tests. Larger threadpools may work fine in perf tests but can cause non-trivial issues where CPU spent more time context switching than actual work due to too many threads.

Reply:
makes sense, making it to halfProcMaxAt10 (same as refresh TP) for now.